### DISCUSSION TITLE
UDP message it's ok only with 127.0.0.1 local syslog server

### DISCUSSION THREAD

**Discussion by marcomoret** [IonxSolutions/serilog-sinks-syslog#72] [Q&A]
Hi, I have a simple problem but is not possibile for me solve it
I start a little test to send a syslog message via UDP on 127.0.0.1:514 local syslog server and the test is ok. If I change 127.0.0.1 IP with real local server IP (10.250.60.50) the syslog message doesn't arrive on local syslog server

The C# test code is simple

private static async Task Main(string[] args)
{
    try
    {
        string system = ConfigurationManager.AppSettings["system"];
        int port = Convert.ToInt32(ConfigurationManager.AppSettings["port"]);
        var log = new LoggerConfiguration()
        .WriteTo.UdpSyslog(host: system, port: port, appName: "my-app", format: SyslogFormat.RFC5424, facility: Facility.Local2, outputTemplate: "{Message}")
        .CreateLogger();

        Console.Write("SEND: This is test message - information");

        //SelfLog.Enable(Console.Error);
        await WriteLogs(log);
    }
    catch (Exception ex)
    {
        Console.Write(ex.ToString());
    }
}

private static async Task WriteLogs(ILogger log)
{
    try
    {
        log.Information("This is test message - information");
        log.Error("This is test message - error");
        //log.Verbose("This is test message - verbose");
        log.Warning("This is test message - warning");
        log.Fatal("This is test message - emergency");

        await Task.Delay(1000);
    }
    catch (Exception ex)
    {
        Console.Write(ex.ToString());
    }
}

**Comment by cocowalla** [comment]
Did you check the firewall on the target machine has UDP/514 open for incoming connections?

**Reply by marcomoret** [reply]
yes, the firewall is stopped

**Reply by cocowalla** [reply]
Oh, I just realised you still want to hit the local server, just by IP instead of loopback.

You should check which NICs your syslog server is bound to - it might only be bound to the loop back adapter.

Also, is your C# app running in a container, or directly on the host?

**Reply by marcomoret** [reply]
host

**Comment by marcomoret** [comment]
sorry, the syslog server was configured to receive only from 127.0.0.1
thanks for all

**Reply by cocowalla** [reply]
No worries, it's an easy thing to miss, and this issue is bound to help others in the future 👍

**Comment by marcomoret** [comment]
my error on server syslog configuration
